python/mlx/nn/layers/activations.py

```diff
@@ -162,6 +163,60 @@ def gelu_fast_approx(x):
     return x * mx.sigmoid(1.773 * x)
 
 
+def glu(x: mx.array) -> mx.array:
```

Comment:
Should we give this an `axis` param?

Reply:
Yes, I thought about adding axis param but at the same time I thought that the last dimension will always be the dimension that will be used for splitting so I found no need to add axis param, but I will add it if you think it will be needed.